### PR TITLE
Add comments to the conversion_hosts table

### PR DESCRIPTION
**Pull request by djberg96**
In my continuing effort to understand things better by documenting them, this is my attempt to document the `conversion_hosts` table.

One question arose which is what is the effective difference between `max_concurrent_tasks` and `concurrent_transformation_limit`.

Feedback and suggestions welcome.

### CODE REVIEW

File: db/migrate/20190213184307_add_comments_to_conversion_hosts_table.rb [hunk 1]
```diff
@@ -0,0 +1,43 @@
+class AddCommentsToConversionHostsTable < ActiveRecord::Migration[5.0]
+  def up
+    change_table_comment :conversion_hosts, "Conversion Hosts"
+
+    change_column_comment :conversion_hosts, :blockio_limit, "The block I/O (disk) limit that the conversion host may use."
```
Comment (bdunne) [inline db/migrate/20190213184307_add_comments_to_conversion_hosts_table.rb]: minor, but alphabetization

Comment (djberg96) [reply db/migrate/20190213184307_add_comments_to_conversion_hosts_table.rb]: Oops, fixed a couple others too, thanks.

**Comment by djberg96** [comment]
@bdunne I wouldn't know how to enforce it. I think that would require some sort of core migration change, wouldn't it?

**Comment by bdunne** [comment]
@djberg96 https://github.com/djberg96/manageiq-schema/pull/1 Let me know what you think

**Comment by djberg96** [comment]
@bdunne Didn't know about automated review. Looks good, merged it, thanks.

**Pull request merged**